src/sidebar/components/group-list-item.js

```diff
@@ -4,18 +4,26 @@ const classnames = require('classnames');
 const propTypes = require('prop-types');
 const { createElement } = require('preact');
 
+const useStore = require('../store/use-store');
 const { orgName } = require('../util/group-list-item-common');
 const { withServices } = require('../util/service-context');
 
-function GroupListItem({ analytics, group, store }) {
+function GroupListItem({ analytics, group }) {
+  const actions = useStore(store => ({
```

Comment:
@robertknight and I discussed this on a call today: this usage right here of `useStore` isn't taking advantage of the re-rendering on callback-state change; it's just using `useStore` to retrieve some actions from the store.